examples/advanced/quantum_volume.ipynb

```diff
@@ -0,0 +1,127 @@
+    "import numpy as np\n",
+    "\n",
+    "# Some set-up variables. Feel free to play with these.\n",
+    "num_repetitions = 11 # This is supposed to be >= 100.\n",
+    "depths = range(2, 5) # The depths and number of qubits\n",
+    "\n",
+    "# Here is the important set-up: The samplers. These are what will be\n",
+    "# run on the generated circuit, and then evaluated.\n",
+    "class SamplerConfig:\n",
+    "    def __init__(\n",
+    "        self,\n",
+    "        *,  # Forces keyword args.\n",
+    "        sampler: cirq.Sampler,\n",
+    "        label: str,\n",
+    "        color: str,\n",
+    "        marker: str = None):\n",
+    "            self.sampler = sampler\n",
+    "            self.label = label\n",
+    "            self.color = color\n",
+    "            self.marker = marker      \n",
+    "            # Keep track of the probabilities.\n",
+    "            self.probabilities = [0] * len(depths)\n",
+    "samplers = [SamplerConfig(\n",
+    "    label = 'Ideal simulation',\n",
+    "    sampler = cirq.Simulator(),\n",
+    "    marker = '+',\n",
+    "    color = 'tab:green',\n",
+    "), SamplerConfig(\n",
+    "    label = 'Noisy simulation',\n",
+    "    sampler = cirq.DensityMatrixSimulator(noise=cirq.ConstantQubitNoiseModel(qubit_noise_gate=cirq.DepolarizingChannel(p=0.005))),\n",
+    "    color = 'tab:red',\n",
+    ")]\n",
+    "\n",
+    "for depth in depths:\n",
+    "    num_qubits = depth # Square matrix.\n",
+    "    print(f\"Running simulation with {num_qubits} qubits and a depth of {depth}\")\n",
+    "    for i in range(num_repetitions):\n",
+    "        print(f\"    Repetition {i + 1} of {num_repetitions}\")\n",
+    "        # Generate a model circuit.\n",
+    "        model_circuit = quantum_volume.generate_model_circuit(\n",
+    "            num_qubits, depth, random_state=np.random.RandomState()\n",
+    "        )\n",
+    "        # Compute its heavy set.\n",
+    "        heavy_set = quantum_volume.compute_heavy_set(model_circuit)\n",
+    "        print(f\"        Heavy Set: {heavy_set}\")\n",
+    "        \n",
+    "        # Run the given samplers over the model circuit.\n",
+    "        for config in samplers:\n",
+    "            probability = quantum_volume.sample_heavy_set(model_circuit, heavy_set, sampler=config.sampler)\n",
+    "            config.probabilities[depth - len(depths)] += probability\n",
+    "            print(f\"        {config.label} HOG probability: {probability}\")\n",
+    "        \n",
+    "        # TODO: The next steps are to compile the model circuit and run it on various devices.\n",
+    "\n",
+    "    # Compute the average performance over the total number of runs.\n",
+    "    for config in samplers:\n",
+    "        config.probabilities[depth - len(depths)] /= num_repetitions\n",
+    "        print(f\"    Average {config.label} HOG probability: {config.probabilities[depth - len(depths)]}\")\n"
```

Comment:
On my screen in the github preview these strings are long enough that they wrap. Reflow long lines.

Reply:
Ran this through yapf, should look better now.